plugins/reporters/evaluated-model/src/main/kotlin/EvaluatedModelMapper.kt

```diff
@@ -410,6 +406,14 @@ internal class EvaluatedModelMapper(private val input: ReporterInput) {
         )
     }
 
+    private fun convertScanResultsForPackage(
```

Comment:
Maybe make this an extension function on `EvaluatedPackage` and just call it "convertScanResult()"?

Although not having an extension function fits better the existing code... but maybe still use the simpler "convertScanResult()" name, as an overload to the function at line 417?

Reply:
> but maybe still use the simpler "convertScanResult()" name

So, what do you think about this?

Reply:
I'd tend to keep this as-is, if this is ok for you.